modules/appeals_api/app/swagger/appeals_api/v2/api_description.md

```diff
@@ -77,6 +80,8 @@ You may check the status of your evidence document upload by using GET `/notice_
 The evidence document upload statuses begin with pending and end with vbms.
 
 Note that until a document status of “received”, “processing”, “success”, or "vbms" is returned, a client cannot consider the document as received by VA. In particular a status of “uploaded” means that the document package has been transmitted, but possibly not validated. Any errors with the document package (unreadable PDF, etc) may cause the status to change to “error”.
+<p>The metadata.json file only supports a limited set of characters within the ascii character space. Refer to the <code>documentUploadMetadata</code> schema for more details.</p>
+<p>See the <code>documentUploadMetadata</code> schema for more details.</p>
```

Comment:
Same here. Just add some newlines to separate the paragraphs instead of `<p>` tags.